src/Coverage/CoverageReporter.php

```diff
@@ -98,6 +98,14 @@ public function php(string $target): void
     public function text(): string
     {
         $text = new Text();
+        if ($this->codeCoverageConfiguration !== null && $this->codeCoverageConfiguration->hasText()) {
+            $text = new Text(
+                50,
+                90,
```

Comment:
PHPUnit gets this values from `coverageReportHtmlType`, can we do the same here and default to 50/90 when those are not set?

Reply:
If I got you right, it's implemented like this, now.